cmd/brew-setup-nginx-conf

```diff
@@ -79,6 +79,15 @@ if `readlink /etc/resolver 2>/dev/null`.chomp != "/usr/local/etc/resolver"
   end
 end
 
+if File.exist? "/etc/pf.anchors/dev.strap"
+  unless system "sudo -n true >/dev/null"
+    puts "Asking for your password to setup launch_socket_server:"
+  end
+  system "sudo rm /etc/pf.anchors/dev.strap"
+  system "brew install launch_socket_server"
```

Comment:
Could just add this to `brewfile` on line 58? Trying to somewhat deprecate manual `brew install` calls in our scripts.